### PR TITLE
Add fiat value instead of balance

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
This PR addresses [BU-228](https://breeztech.atlassian.net/browse/BU-228).

- Balance title is removed from wallet dashboard.
- Fiat equivalent of balance is shown under balance and users can switch between fiat currencies by clicking on this field.
- This field is only shown if the amount is bigger than minimal value, e.g. "< $0.01"
- Clicking on the fiat value will look up the next fiat currency that's above minimal value.

### CODE REVIEW

File: lib/routes/home/wallet_dashboard.dart [hunk 1]
```diff
@@ -114,7 +102,38 @@ class WalletDashboardState extends State<WalletDashboard> {
...
+                        (widget._accountModel.fiatConversionList.isNotEmpty ||
+                            widget._accountModel.fiatCurrency != null) &&
+                        isAboveMinAmount(widget._accountModel.fiatCurrency)
+                    ? FlatButton(
+                        onPressed: () {
+                          var nextFiatCurrencyIndex = widget
+                                  ._accountModel.fiatConversionList
+                                  .indexOf(widget._accountModel.fiatCurrency) +
+                              1;
+                          findNextValidFiatConversion(nextFiatCurrencyIndex %
```
Comment (roeierez) [inline lib/routes/home/wallet_dashboard.dart:120]: The name of `findNextValidFiatConversion` implies read-only. Better to keep it read-only and call the `_onFiatCurrencyChange` callback here.

File: lib/routes/home/wallet_dashboard.dart [hunk 2]
```diff
@@ -114,7 +102,38 @@ class WalletDashboardState extends State<WalletDashboard> {
...
-          )
+          ),
+          Positioned(
+              top: 70 - BALANCE_OFFSET_TRANSITION * widget._offsetFactor,
+              child: Center(
+                child: widget._accountModel != null &&
+                        !widget._accountModel.initial &&
+                        (widget._accountModel.fiatConversionList.isNotEmpty ||
+                            widget._accountModel.fiatCurrency != null) &&
+                        isAboveMinAmount(widget._accountModel.fiatCurrency)
```
Comment (roeierez) [inline lib/routes/home/wallet_dashboard.dart:113]:
```suggestion
                        isAboveMinAmount(widget._accountModel?.fiatCurrency)
```

File: lib/routes/home/wallet_dashboard.dart [hunk 3]
```diff
@@ -114,7 +102,38 @@ class WalletDashboardState extends State<WalletDashboard> {
...
+                    ? FlatButton(
+                        onPressed: () {
+                          var nextFiatCurrencyIndex = widget
+                                  ._accountModel.fiatConversionList
+                                  .indexOf(widget._accountModel.fiatCurrency) +
+                              1;
```
Comment (roeierez) [inline lib/routes/home/wallet_dashboard.dart:119]: findNextValidFiatConversion is a method of this class and has access to the widget._accountModel. I think it is better to move all the logic there and keep this UI section simpler.

File: lib/routes/home/wallet_dashboard.dart [hunk 4]
```diff
@@ -130,4 +149,23 @@ class WalletDashboardState extends State<WalletDashboard> {
       },
     );
   }
+
+  void findNextValidFiatConversion(int nextIndex, {int retries = 0}) {
```
Comment (roeierez) [inline lib/routes/home/wallet_dashboard.dart:153]: Maybe a for-loop will read simpler than a recursive function here?

Comment (erdemyerebasmaz) [reply lib/routes/home/wallet_dashboard.dart]: https://github.com/breez/breezmobile/pull/338/commits/eb9e3c34ea346715bf0efa1572f535f5428933ec

File: lib/routes/home/wallet_dashboard.dart (hunk 3 above)
Comment (erdemyerebasmaz) [reply lib/routes/home/wallet_dashboard.dart]: This logic is moved to findNextValidFiatConversion(nextValidFiatConversion) method https://github.com/breez/breezmobile/pull/338/commits/eb9e3c34ea346715bf0efa1572f535f5428933ec

File: lib/routes/home/wallet_dashboard.dart (hunk 2 above)
Comment (erdemyerebasmaz) [reply lib/routes/home/wallet_dashboard.dart]: Done https://github.com/breez/breezmobile/pull/338/commits/334e7b2861d9a43a51c30ad86a730685a70249f0

**Pull request merged**